prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Contact Us</title>
+  </head>
+  <body>
+    <h1>Contact Us</h1>
+    <div>
+      <a
+        href="file:///Users/chorongshin/repos/prep-1121-code-solutions/prep-html-links/index.html"
+        >Click here to go to the home page</a
+      >
```

Comment:
Try to use a relative URL for your `href`. Otherwise, your code won't work on anyone else's computer, since this URL has your username in it. For example:

```html
<a href="../index.html">Click here to go to the home page</a>
```